packages/react-make-styles/bundle-size/makeStyles.fixture.js

```diff
@@ -0,0 +1,7 @@
+import { __styles, mergeClasses } from '@fluentui/react-make-styles';
```

Comment:
**q:** will this work if we use relative imports (preferable as it's already scoped in same package)

-> meaning `import { __styles, mergeClasses } from '../src';`

Reply:
Nope, it will not be possible as it tool runs over built artifacts:

> relies on build artifacts (i.e. requires build first) as we want to test the same output as customers will consume

Reply:
got it, ty.

this keeps me wondering, how would  this work with some future adjustment that I (you as well) mentioned few times regarding build output ([ref](https://github.com/microsoft/fluentui/pull/17298#discussion_r593089601)).

#### context

- having single `/dist` in root

```
|- dist/
|- |- <package-a>/
|- |- |- index.js
|- |- |- index.umd.js
|- |- |- index.d.ts
|- |- |- package.json

|- |- <package-b>/
```

meaning we'd need to do following (for local fixtures):

```
import { __styles, mergeClasses } from '../../../dist/react-make-styles';
```

will this work ?

Reply:
> this keeps me wondering, how would this work with some future adjustment that I (you as well) mentioned few times regarding build output ([ref](https://github.com/microsoft/fluentui/pull/17298#discussion_r593089601)).

Good call. 



> will this work ?

Yes, this will work, but I would rather try to setup aliases on Webpack level to `/dist`. WDYT?

Reply:
interesting idea, ok lets iterate on it as we go. (I'd like to keep only 1 source of truth for aliases -> tsconfig.base.json )